examples/store/db/index.ts

```diff
@@ -1,6 +1,23 @@
 import {PrismaClient} from '@prisma/client'
 export * from '@prisma/client'
 
-const prisma = new PrismaClient()
+let prisma
+
+declare global {
+  namespace NodeJS {
+    interface Global {
+      prisma: PrismaClient
+    }
+  }
+}
```

Comment:
Thanks for these updates!

One thing, let's remove this because we don't want people to actually use `global.prisma`.

I think switching to `global["prisma"]` will fix the TS warning without needing the type declaration.

Probably also need `let prisma: PrismaClient`

Reply:
ok let me try that instead, thanks for the tips!